src/QueryType/Update/Query/Command/Raw.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Solarium\QueryType\Update\Query\Command;
+
+use Solarium\QueryType\Update\Query\Query as UpdateQuery;
+
+/**
+ * Update query raw XML command.
+ *
+ * @see http://wiki.apache.org/solr/UpdateXmlMessages
+ */
+class Raw extends AbstractCommand
```

Comment:
As mentioned above the class should be named explicit `RawXML` as there're raw JSON commands, too.